design-doc-template.adoc

```diff
@@ -82,73 +94,98 @@ __<List the installation types that are relevant for the features and remove any
+should remove this section.
 
-__<For any existing requirements from the feature being promoted that are being changed or removed, describe the change.>__
+For any existing requirements from the feature being promoted that are being changed or removed, describe the change.
+////
 
 
 === Non-Requirements
 
-__<Use this section to explicitly discuss things that readers might think are required but which are not required.>__
+////
+Use this section to explicitly discuss things that readers might think are required but which are not required.
+////
 
 === Future Work
 
-__<Use this section to discuss requirements that are not addressed by this proposal but which may be addressed in later proposals.>__
+////
+Use this section to discuss requirements that are not addressed by this proposal but which may be addressed in later proposals.
+////
 
 == Backwards Compatibility
 
-__<Does this enhancement affect backwards compatibility with previously released versions of WildFly? Can the identified
-incompatibility be avoided?>__
+////
+Does this enhancement affect backwards compatibility with previously released versions of WildFly? Can the identified
+incompatibility be avoided?
+////
 
 === Default Configuration
 
-__<Does the proposed work change the default value of any current configuration attributes? Does it change the
-configuration generated by any current Galleon layers?>__
+////
+Does the proposed work change the default value of any current configuration attributes? Does it change the
+configuration generated by any current Galleon layers?
+////
 
 === Importing Existing Configuration
 
-__<Does the proposed work affect the ability to run WildFly running an existing configuration? Is there anything else
-about the proposed work that may require changes to the WildFly server migration tool?>__
+////
+Does the proposed work affect the ability to run WildFly running an existing configuration? Is there anything else
+about the proposed work that may require changes to the WildFly server migration tool?
+////
 
 === Deployments
 
-__<Does this feature change the behavior of deployments in incompatible ways? If yes, please detail what is the deployment
-issue observed when no change is done, and what is the change needed to solve the deployment issue.>__
+////
+Does this feature change the behavior of deployments in incompatible ways? If yes, please detail what is the deployment
+issue observed when no change is done, and what is the change needed to solve the deployment issue.
+////
 
 === Interoperability
 
-__<Is this feature impacting interoperability?>__
+////
+Is this feature impacting interoperability?
+////
 
 == Implementation Plan
 
-__<This section is optional. If you have a complex feature which cannot be delivered all in one go, suggest the strategy.>__
+////
+This section is optional. If you have a complex feature which cannot be delivered all in one go, suggest the strategy.
```

Comment:
```suggestion
This section is optional. If you have a complex feature which cannot be delivered all in one go, suggest the strategy. Otherwise, delete this section.
```